pkg/rpc/rpc.go

```diff
@@ -0,0 +1,61 @@
+package rpc
+
+import (
+	"fmt"
+
+	rpcclient "github.com/cometbft/cometbft/rpc/client/http"
+	"github.com/cosmos/cosmos-sdk/client"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"google.golang.org/grpc"
+
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	fungibletypes "github.com/zeta-chain/zetacore/x/fungible/types"
+	lightclienttypes "github.com/zeta-chain/zetacore/x/lightclient/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+// Clients contains RPC client interfaces to interact with zetacored
+type Clients struct {
+	AuthorityClient authoritytypes.QueryClient
+	CctxClient      crosschaintypes.QueryClient
```

Comment:
```suggestion
	CrosschainClient      crosschaintypes.QueryClient
```
This is actually a wrong naming we have in the E2E suite